packages/charts/vaadin-chart.js

```diff
@@ -1,3 +1,2 @@
-import './vaadin-chart-default-theme.js';
-import './src/vaadin-chart.js';
-export * from './src/vaadin-chart.js';
+import './theme/lumo/vaadin-chart.js';
+export * from './theme/lumo/vaadin-chart.js';
```

Comment:
This differs from other components in that it exports from under theme/lumo. For example, the corresponding file in [text field exports from src](https://github.com/vaadin/web-components/blob/master/packages/text-field/vaadin-text-field.js#L3) and the [files under theme don't export anything](https://github.com/vaadin/web-components/blob/master/packages/text-field/theme/lumo/vaadin-text-field.js)

Reply:
I believe this was intentional, but I'm going to have to confirm with @alvarezguille

Reply:
good catch, I don't think there's a reason for this to be different to other components

Reply:
I changed it so it exports the file from `src`.

Reply:
The second part still wasn't changed https://github.com/vaadin/web-components/pull/3006/files#diff-de2d3dfcdecdc9ac012e815f924e8d99293af73e3eb01a1ad0082773fe6b7408R4